.github/workflows/codeql-analysis.yml

```diff
@@ -43,4 +43,4 @@ jobs:
     #   make release
 
     - name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@v2
```

Comment:
`v2` isn't deprecated now, but it will be deprecated soon, so I figured I might as well go ahead and update these now.

Reply:
Ok